utils/fix_rules.py

```diff
@@ -702,6 +730,31 @@ def sort_subkeys(args, product_yaml):
     return int(len(results) > 0)
 
 
+@command("sort_prodtypes", "sorts the products in prodtype")
```

Comment:
Nit: “sorts products in the prodtype” or “sorts the products in the prodtype”. My 2c but I think the latter sounds best? As written, to me, it sounds like it is lacking a ‘the’.